eth2-ssv/storage/collections/ibft_storage.go

```diff
@@ -0,0 +1,146 @@
+package collections
```

Comment:
Would recommend a schema migration pattern in place to make it easier to upgrade certain parts of the DB that might become bottlenecks in production. See https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/db/kv/migration.go